README.md

```diff
@@ -18,6 +18,10 @@ major.minor version of dbt it's for so `1.3.45` release includes the latest patc
 Note that the patch version *will not* match the patch version for dbt-core or any other package as it will increment 
 with a change to any of the included python packages.  
 
+#### EOL Packages
```

Comment:
This isn't feedback on this PR as much as an open-ended question resulting from this PR. Why wouldn't we just publish these to PyPI then? And if we don't want to maintain it post EOL (which feels like the definition of EOL), why would we incorporate security patches at all? Wouldn't this just result in users pointing their projects to the GH repo instead of PyPI (much like we're doing here)?